### DISCUSSION TITLE
Web belajar dari sesi 1 dan 2

### DISCUSSION THREAD

**Discussion by Abdilluv** [deaafrizal/forum#24] [My Projects]
https://myweb.abdillahfajar22.repl.co/index.html

**Comment by Abdilluv** [comment]
makasih ilmunya cuy

**Comment by deaafrizal** [comment]
mantep sekali brother! 👌

**Comment by Destaanugra1** [comment]
![ai](https://github.com/deaaprizal/forum/assets/152362800/972793ba-50a8-4245-ba72-2d20912166a8)
<div>
      <h1>Paling</h1>
      {anime.data.map(data => {
        return <AnimeList title={data.title} images={data.images.webg.image_url} />
                                                                                                            ^
      })}
    </div>
  )
  kenapa error di situ bang
